packages/generator-langium/templates/web/scripts/bundleVscodeApi.js

```diff
@@ -0,0 +1,8 @@
+import 'vscode/default-extensions/theme-defaults';
+import { MonacoEditorLanguageClientWrapper } from 'monaco-editor-wrapper';
+import { addMonacoStyles } from 'monaco-editor-wrapper/styles';
+
+export {
+    addMonacoStyles,
+    MonacoEditorLanguageClientWrapper
+}
```

Comment:
Thanks for the hint. It can be just that (fixed in next commit ⬇️):
```javascript
import 'vscode/default-extensions/theme-defaults';
export * from './bundleClassic.js';
```